index.bs

```diff
@@ -314,7 +317,7 @@ A [=screen/screen area=] has a <dfn>height</dfn>, which is the number of [=scree
 
 Note: The grid size is usually expressed as <[=screen/width=]>×<[=screen/height=]>. For example, a 1920×1080 screen area has a grid with a [=screen/width=] of 1920 pixels and a [=screen/height=] of 1080 pixels.
 
-Note: As specified in [[CSSOM-VIEW#web-exposed-screen-information]], user agents may choose to hide information about the screen of the output device in order to protect the user’s privacy. In this case, the [=screen/screen area=] may be equal to that of the [viewport](https://drafts.csswg.org/css2/#viewport).
+Note: As specified in [[CSSOM-VIEW-1#web-exposed-screen-information]], user agents may choose to hide information about the screen of the output device in order to protect the user’s privacy. In this case, the [=screen/screen area=] may be equal to that of the [viewport](https://drafts.csswg.org/css2/#viewport).
```

Comment:
@tidoust 
Hi Francois,

On my recent PR, there is a publishing error related to this change in your PR:

> FATAL ERROR: Couldn't find section '#web-exposed-screen-information' in spec 'cssom-view-1':
>       [[CSSOM-VIEW-1#web-exposed-screen-information]]

Can you advise on how to fix this?

Reply:
Dug into this a bit..

```bash
bikeshed refs --spec=cssom-view-1 | grep "#web-" -B 3 -A 3
```

Doesn't show any references named `#web-exposed-screen-information`, but it's clearly on the [draft](https://drafts.csswg.org/cssom-view-1#web-exposed-screen-information). CSSOM-VIEW is showing a [fatal bikeshed error](https://drafts.csswg.org/bikeshed/cssom-view/) which could be why bikeshed does not have references from that spec. I filed an [issue](https://github.com/w3c/csswg-drafts/issues/7991) with cssom-view-1 to resolve the bikeshed error.

Reply:
Looks like this is a bikeshed bug: https://github.com/tabatkins/bikeshed/issues/2379

Reply:
Yes, this was due to a bug that temporarily emptied Bikeshed's headings database.

Bikeshed generation still fails but that's because the spec attempts to link to `window-placement` whereas that term is no longer defined, see #116 for a proposed fix.